src/structlog/stdlib.py

```diff
@@ -477,27 +574,31 @@ class ProcessorFormatter(logging.Formatter):
 
     def __init__(
         self,
-        processor,
-        foreign_pre_chain=None,
-        keep_exc_info=False,
-        keep_stack_info=False,
-        logger=None,
-        pass_foreign_args=False,
-        *args,
-        **kwargs
-    ):
+        processor: Processor,
+        foreign_pre_chain: Optional[List[Processor]] = None,
+        keep_exc_info: bool = False,
+        keep_stack_info: bool = False,
+        logger: logging.Logger = None,
```

Comment:
Not sure if that will impact types somehow, but you use mix of `Optional[X] = None`, and `X = None`, in later one mypy assume Optional implicitly, you can disable that with `--no-implicit-optional` config option.

Reply:
It's also possible for implicit optional to "leak" out to downstream users - I discovered this via https://github.com/HypothesisWorks/hypothesis/pull/2613

Reply:
Whenever I think I'm done, someone unearths another mypy option that adds another ~20 errors. 😅

Should be fixed in ab10696865d0a261dafc758b1c857c9d00d96128